app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <color name="primary">@color/grey300</color>
+    <color name="primaryVariant">@color/black</color>
+    <color name="onPrimary">@color/black</color>
+
+    <color name="secondary">@color/orange300</color>
+    <color name="secondaryVariant">@color/orange700</color>
+    <color name="onSecondary">@color/white</color>
```

Comment:
please use default android color for white @android:color/white. Do this to other files